docs/example-usage.ipynb

```diff
@@ -0,0 +1,1478 @@
+       "    valid_min:  0.0\n",
+       "    valid_max:  1.0</pre><div class='xr-wrap' style='display:none'><div class='xr-header'><div class='xr-obj-type'>xarray.DataArray</div><div class='xr-array-name'>'albedo'</div><ul class='xr-dim-list'><li><span>phony_dim_0</span>: 16893</li></ul></div><ul class='xr-sections'><li class='xr-section-item'><div class='xr-array-wrap'><input id='section-2d388e93-7cda-4d85-8028-71ef333ef92a' class='xr-array-in' type='checkbox' checked><label for='section-2d388e93-7cda-4d85-8028-71ef333ef92a' title='Show/hide data repr'><svg class='icon xr-icon-database'><use xlink:href='#icon-database'></use></svg></label><div class='xr-array-preview xr-preview'><span>...</span></div><div class='xr-array-data'><pre>[16893 values with dtype=float32]</pre></div></div></li><li class='xr-section-item'><input id='section-55be37d6-917d-4d44-bf21-451f683d2ca2' class='xr-section-summary-in' type='checkbox' disabled ><label for='section-55be37d6-917d-4d44-bf21-451f683d2ca2' class='xr-section-summary'  title='Expand/collapse section'>Coordinates: <span>(0)</span></label><div class='xr-section-inline-details'></div><div class='xr-section-details'><ul class='xr-var-list'></ul></div></li><li class='xr-section-item'><input id='section-1f6180fb-b0c6-4757-9ab5-549fadff9780' class='xr-section-summary-in' type='checkbox' disabled ><label for='section-1f6180fb-b0c6-4757-9ab5-549fadff9780' class='xr-section-summary'  title='Expand/collapse section'>Indexes: <span>(0)</span></label><div class='xr-section-inline-details'></div><div class='xr-section-details'><ul class='xr-var-list'></ul></div></li><li class='xr-section-item'><input id='section-93222571-bfee-4b80-adf8-27bca797da06' class='xr-section-summary-in' type='checkbox'  checked><label for='section-93222571-bfee-4b80-adf8-27bca797da06' class='xr-section-summary' >Attributes: <span>(3)</span></label><div class='xr-section-inline-details'></div><div class='xr-section-details'><dl class='xr-attrs'><dt><span>long_name :</span></dt><dd>Diffuse reflecting power of the Earth&amp;apos;s surface within the grid cell.</dd><dt><span>valid_min :</span></dt><dd>0.0</dd><dt><span>valid_max :</span></dt><dd>1.0</dd></dl></div></li></ul></div></div>"
+      ],
+      "text/plain": [
+       "<xarray.DataArray 'albedo' (phony_dim_0: 16893)> Size: 68kB\n",
+       "[16893 values with dtype=float32]\n",
+       "Dimensions without coordinates: phony_dim_0\n",
+       "Attributes:\n",
+       "    long_name:  Diffuse reflecting power of the Earth&apos;s surface within t...\n",
+       "    valid_min:  0.0\n",
+       "    valid_max:  1.0"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "in_datatree['Soil_Moisture_Retrieval_Data']['albedo']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f609c11a-173c-4a23-aab9-57bb38eab134",
+   "metadata": {},
+   "source": [
+    "### Locating grid values\n",
```

Comment:
I like the explanation here showing what the service is doing with clear examples. Nice.